labs/iot/gps-lab/steps/set-up-azure-maps.md

```diff
@@ -0,0 +1,23 @@
+# Set Up Azure Map
+
+In this step you will set up Azure Map.
+
+## Steps
+
+1. Go to [Azure portal]('https://portal.azure.com') and login to your subscription. 
+2. Create a resource for Azure Maps  ![Azure Map](../images/azure-map-create-1.png)
+3. Select Azure Map![Azure Map](../images/azure-map-create-2.png)
+4. Note down the Primary Key and in [application.js](../server/code/static/js/application.js) and replace <Azure_Map_Subscription_Key> with the primary key
+![Azure Map](../images/azure-map-create-3.png)
+    ```js
+    //Initialize a map instance.
+    map = new atlas.Map('myMap', {
+       ...
+        authOptions: {
+            authType: 'subscriptionKey',
+            subscriptionKey: '<Azure_Map_Subscription_Key>'
+        }
+    });
+    ```
```

Comment:
Once this has been updated, is there a way to test it works? Can we have a validation step where they run the web app and see the maps working?

Reply:
If this is wrong they will not see the map. I have updated the document to include this.